help/en/package/jmri/jmrit/conditional/ConditionalListEditor.shtml

```diff
@@ -285,7 +285,27 @@
       will depend on the conditional name selection method setting.
       See <a href="../beantable/LogixTable.shtml#logixOptions">
       Conditional Name Selection Method</a>.</p>
-      
+
+      <ul>
+         Note:<br>
+         As an advanced feature, the name of an entity can be entered as
+         an "indirect reference" to a memory location.  By that we mean, 
+         rather than naming the action directly, the value of a memory  
+         location is used for the name of the action. To do this you enter  
+         the name of the memory entity preceded with the "@" symbol.
```

Comment:
Not a comment about this PR, but a question about Logix:

Where is this handled in the code? I'm currently working on code for importing Logixs to LogixNGs, so I need to support import of Conditionals that uses this feature.